config/config_hera_gnu.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# Compiler/MPI combination
+export HPC_COMPILER="gnu/9.2.0"
+export HPC_MPI="mpich/3.3.2"
+export HPC_PYTHON="miniconda3/3.7.3"
+
+# Build options
+export USE_SUDO=N
+export PKGDIR=pkg
+export LOGDIR=log
+export OVERWRITE=N
+export NTHREADS=8
+export   MAKE_CHECK=N
+export MAKE_VERBOSE=N
+export   MAKE_CLEAN=N
+export DOWNLOAD_ONLY=N
+export STACK_EXIT_ON_FAIL=Y
+export WGET="wget -nv"
+
+# Load these basic modules for Hera
+module purge
+module load cmake/3.16.1
+
+# To find miniconda3/3.7.3
+module use /scratch1/NCEPDEV/nems/emc.nemspara/soft/modulefiles
```

Comment:
Is there something wrong with:
```
module use /contrib/miniconda3/modulefiles
```

It provides a miniconda3 modulefile for version 4.5.12?

I have been using that for creating a virtual env. for JEDI work.

Reply:
There is no virtual environment in this setup, everything is part of the base environment of this installation. Making the change as you suggested will not have all the dependencies. If someone wants to change that, then that's great, but it won't work right away.

Reply:
And that is not appropriate.
Is emc.nemspara going to provide the python packages for all developers?
The correct way to do this is to use base python, and create python virtual environments for the applications.

Reply:
Please go ahead and update this with the next ufs-weather-model PR. The problem has been discussed for days and held back the GOCART PR, nobody was willing to take action.

Note that using the conda init, activate and deactivate functionality in many cases does not work, one still has to manually set PYTHONPATH, PATH, LD_LIBRARY_PATH or create modulefiles to do that.

Reply:
There is no `conda init`, `conda activate`, `conda deactivate` in the case of python virtual environments. 
These "new" GOCART dependencies came out of nowhere.  There was a previous PR to hpc-stack that added GOCART dependencies.

If you give me a list of which python packages that are required for the UFS weather model, I can try and create a PR with those as a python virtual env. and its associated modulefile.

Reply:
That would be great. What we need is Python3 pandas and netCDF4 (these were the ones that I had to install), and ecflow (unless we pick that from somewhere else). That's all I think.

Reply:
We could combine switching to the virtual env with rolling out hpc-stack v.1.2.0 that then uses the same miniconda3. Thanks for helping with this!